lib/node/rpc.js

```diff
@@ -2074,21 +2091,46 @@ class RPC extends RPCBase {
       if (!secp256k1.publicKeyVerify(key))
         throw new RPCError(errs.INVALID_ADDRESS_OR_KEY, 'Invalid key.');
 
+      keyString += items.str(i) + ',';
       keys[i] = key;
     }
 
-    const script = Script.fromMultisig(m, n, keys, true, false);
+    keyString = keyString.slice(0, -1);
 
-    if (script.getSize() > consensus.MAX_SCRIPT_PUSH) {
-      throw new RPCError(errs.VERIFY_ERROR,
-        'Redeem script exceeds size limit.');
-    }
+    const isWitness = outputType !== outputTypes.LEGACY;
+    const script = Script.fromMultisig(m, n, keys, false, isWitness);
```

Comment:
I think the better way to get this script is `const script = subdesc.generateScripts()[0];` instead of using Script.fromMultisig(...). What do you think?